pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -124,7 +129,16 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 			plugins = append(plugins, saAdmitter)
 
 		default:
-			plugin := admission.InitPlugin(pluginName, kubeClient, server.AdmissionControlConfigFile)
+			configFile := server.AdmissionControlConfigFile
+			pluginConfig := options.KubernetesMasterConfig.AdmissionConfig.PluginConfig
+
+			if cfg, hasConfig := pluginConfig[pluginName]; hasConfig {
```

Comment:
the difference here is that I have a default config file (line 132) that I don't want to overwrite if no config is present. The GetPluginConfig function would return a blank name if the config is not there.